templates/accessibility.html

```diff
@@ -5,71 +5,82 @@
 {% block copydoc %}https://docs.google.com/document/d/1_cCvuHSwS9i0pzD_4WHDFoTenVAfZVr9qGxG-rSHHGY/edit{% endblock %}
```

Comment:
This page has a copydoc: https://docs.google.com/document/d/1_cCvuHSwS9i0pzD_4WHDFoTenVAfZVr9qGxG-rSHHGY/edit#heading=h.k45g1u10i0ni

I think we should update that as well to the new content.